styles/elements/inputs/button/_mixins.scss

```diff
@@ -48,8 +49,7 @@
   // Pressed state
   &:active,
   &.is-active, // the .is-active class is to be used when the button needs to be manually pressed, like for dropdowns
-  .open &.dropdown-toggle, // TODO: remove this once dropdowns are replaced in iverson
-  &.ice-dropdown-toggle.is-open {
+  .open &.dropdown-toggle { // TODO: remove this once dropdowns are replaced in iverson
```

Comment:
Are we keeping a list of these, or should we create a few "codes" to add to comments so we can easily find what we're looking for when it's time to do this?

Reply:
Keeping a list might be too tedious I think, easy to forget to update the list, etc.
But I've just been using "TODO:" so I can search that later. If you want to come up with codes thats fine. If its an ember version change I usually do "TODO: Ember 1.13" etc for example.